app.rb

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env ruby
+
+require './student'
+require './teacher'
+require './book'
+require './rental'
+
+class App
+  def initialize
+    @books = []
+    @person = []
+    @rentals = []
+  end
+# rubocop:disable ...
+  def select_opt
+    # rubocop:disable ...
+    option = gets.chomp.to_i
+    case option
+    when 1 then list_books
+    when 2 then list_people
+    when 3 then create_person
```

Comment:
[REQUIRED] Just like the previous reviewers I still get an error when I try to create a person